.travis.yml

```diff
@@ -24,17 +23,14 @@ dist: bionic
 
 language: java
 jdk:
-  - openjdk8
-
-# disabling cache for /home/travis/.m2/repository/org/apache/hive/hive-jdbc/3.0.0-SNAPSHOT/hive-jdbc-3.0.0-SNAPSHOT-standalone.jar (Permission denied)
-#cache:
-#  directories:
-#  - $HOME/.m2
+  - openjdk11
 
 env:
   MAVEN_SKIP_RC=true
   MAVEN_OPTS="-Xmx2g"
 
 install: skip
 
-script: mvn clean install -DskipTests -q -Pitests
+before_script: mvn --version
```

Comment:
I think we should just remove `travis` entirely...no need to do this 2 times

Reply:
What is the difference between the two exactly?  I like that I can easily create per-branch build envs with Travis.

Reply:
travis doesn't run tests - it just compiles

Reply:
Ya, sorry, I mean from a feature standpoint : Travis v.s. Jenkins